v3/src/components/axis-graph-shared.ts

```diff
@@ -3,6 +3,8 @@ import {AxisPlaces} from "./axis/axis-types"
 export const GraphPlaces = [...AxisPlaces, "yPlus", "plot", "legend"] as const
 export type GraphPlace = typeof GraphPlaces[number]
 
+export type GraphExtentsPlace = GraphPlace | "banners"
```

Comment:
Not sure this is the perfect spot for this, but I couldn't think of a more appropriate location.

Reply:
My inclination would be to define it in `data-display-layout.ts`. It has no meaning outside of the layout context. As a separate matter, I noticed in the code below that we return a value for `yPlus` along with a comment indicating that it is unused. It should be possible to do this instead:
```typescript
export type GraphExtentsPlace = Exclude<GraphPlace | "banners", "yPlus">
```
where `Exclude` comes from the `type-fest` library, and then the extraneous property wouldn't be necessary.